testament/important_packages.nim

```diff
@@ -1,159 +1,156 @@
 ##[
-## note
+## note 1
 `useHead` should ideally be used as the default but lots of packages (e.g. `chronos`)
 don't have release tags (or have really old ones compared to HEAD), making it
 impossible to test them reliably here.
 
 packages listed here should ideally have regularly updated release tags, so that:
 * we're testing recent versions of the package
 * the version that's tested is stable enough even if HEAD may occasionally break
-]##
 
+## note 2: D20210308T165435:here
```

Comment:
note to reviewer: make sure you agree with that otherwise this needs further discussion
(/cc @cheatfate)